src/sonic-yang-models/yang-models/sonic-buffer-profile.yang

```diff
@@ -0,0 +1,67 @@
+module sonic-buffer-profile {
+	namespace "http://github.com/Azure/sonic-buffer-profile";
+	prefix bpf;
+
+	import sonic-buffer-pool {
+		prefix bpl;
+	}
+
+	organization
+		"SONiC";
+
+	contact
+		"SONiC";
+
+	description
+		"SONIC BUFFER PROFILE";
+
+	revision 2019-05-15 {
+		description
+			"Initial revision.";
+	}
+
+
+	container sonic-buffer-profile {
+
+		container BUFFER_PROFILE {
+
+			list BUFFER_PROFILE_LIST {
+				key "name";
+
+				leaf name {
+					type string;
+				}
+
+				leaf static_th {
+					type uint64;
+				}
+
+				leaf dynamic_th {
+					type int64;
+				}
```

Comment:
is there way to express the constraint that static_th and dynamic_th as mutually exclusive?

Reply:
We can try with count(field-name) = 0 or 1 condition in a must statement.

Reply:
I will try with choice statement.

Reply:
Tried with choice but hitting error. Please refer comments from Praveen below.